clients/src/main/java/org/apache/kafka/clients/admin/CreateDelegationTokenOptions.java

```diff
@@ -53,12 +53,23 @@ public Optional<KafkaPrincipal> owner() {
         return Optional.ofNullable(owner);
     }
 
-    public CreateDelegationTokenOptions maxlifeTimeMs(long maxLifeTimeMs) {
-        this.maxLifeTimeMs = maxLifeTimeMs;
+    @Deprecated
```

Comment:
I think we should add a javadoc to indication this.
```suggestion
    /**
     * @deprecated since 4.0 and should not be used any longer.
     */
    @Deprecated
```